utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -104,6 +104,16 @@ Describe 'File/folder tests' -Tag 'Modules' {
             $file.Name | Should -BeExactly 'README.md'
         }
 
+        It '[<moduleFolderName>] A [` version.json `] file must only have a major & minor version.' -TestCases ($moduleFolderTestCases | Where-Object { Test-Path (Join-Path $_.moduleFolderPath 'version.json') }) {
```

Comment:
Recommend disabling workflows for this and run the automation account manuall after the merge